package.json

```diff
@@ -7,6 +7,7 @@
     "@angular/core": "^16.1.0",
     "@angular/platform-browser": "^16.1.0",
     "@angular/platform-browser-dynamic": "^16.1.0",
+    "@angular/router": "^16.1.0",
```

Comment:
This project shouldn't concern it's self with the router.

Reply:
it's used by the sample